### DISCUSSION TITLE
ScriptReference/Editor.OnPreviewGUI

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#45] [Notes]
# ScriptReference/Editor.OnPreviewGUI

https://docs.unity3d.com/ScriptReference/Editor.OnPreviewGUI.html

**Comment by vertxxyz** [comment]
If you do implement this method you should also implement [`HasPreviewGUI`](https://docs.unity3d.com/ScriptReference/Editor.HasPreviewGUI.html).
